lib/ubus.c

```diff
@@ -2438,10 +2452,25 @@ uc_ubus_channel_connect(uc_vm_t *vm, size_t nargs)
 }
 
 
+static uc_value_t *
+uc_ubus_exception_handler_set(uc_vm_t *vm, size_t nargs)
+{
+	uc_value_t *arg = uc_fn_arg(0);
+
+	if (arg && !ucv_is_callable(arg))
+		return NULL;
+
+	uc_vm_registry_set(vm, "ubus.ex_handler", ucv_get(arg));
+
+	return ucv_boolean_new(true);
+}
+
+
 static const uc_function_list_t global_fns[] = {
 	{ "error",			uc_ubus_error },
 	{ "connect",		uc_ubus_connect },
 	{ "open_channel",	uc_ubus_channel_connect },
+	{ "exception_handler_set", uc_ubus_exception_handler_set },
```

Comment:
To stay in line with the mostly one word function naming I would suggest `trap()` or `guard()` as method names.